docs/content/guides/dagster/code-references.mdx

```diff
@@ -65,7 +65,7 @@ Dagster's dbt integration can automatically attach references to the SQL files b
 
```

Comment:
```
A link to the asset's source in `with_source_code_references.py` will then be visible in the **Asset Catalog** view in the Dagster UI:
```

There's no reference to `with_source_code_references.py` in the above prose/code.

Reply:
Also that image is way too big